src/Core/Mesh/TopologicalTriMesh/TopologicalMesh.hpp

```diff
@@ -98,6 +100,130 @@ class RA_CORE_API TopologicalMesh : public OpenMesh::PolyMesh_ArrayKernelT<Topol
     inline const std::vector<OpenMesh::HPropHandleT<Vector3>>& getVector3PropsHandles() const;
     inline const std::vector<OpenMesh::HPropHandleT<Vector4>>& getVector4PropsHandles() const;
     ///@}
+    
+    /// \name Dealing with normals
```

Comment:
prefer /** ... */ for multiline documentation. Easyer to maintain and to isolate ...